package.json

```diff
@@ -307,6 +307,7 @@
     "fast-levenshtein": "^2.0.6",
     "fine-uploader": "^5.16.2",
     "foundation-sites": "^5.5.3",
+    "he": "^1.2.0",
```

Comment:
The author of this library really didn't put a lot into naming this one lol. I think we need to run `yarn install` to after this though

Reply:
Ah for the yarn.lock you mean? Good idea

Reply:
Hmm, so I ran `yarn install` and it's not showing up in the `yarn.lock`. So then I deleted the `yarn.lock` and ran it, still no sign of an updated `he` within it. Very odd

Reply:
@kelsonic I can see it in mine when I pull the branch. Searching for `"he` didn't work, but `he@` does so I think you're good:

<img width="833" alt="Screen Shot 2021-03-18 at 10 20 33 AM" src="https://user-images.githubusercontent.com/459581/111641695-c4879080-87d3-11eb-8b85-50d4441e566e.png">

Reply:
awesome, thanks!